experimental/packages/opentelemetry-instrumentation-http/src/utils.ts

```diff
@@ -445,6 +474,24 @@ export const getIncomingRequestAttributes = (
   return Object.assign(attributes, httpKindAttributes, options.hookAttributes);
 };
 
+/**
+ * Returns incoming request Metric attributes scoped to the request data
+ * @param {SpanAttributes} spanAttributes the span attributes
+ * @param {{ component: string }} options used to pass data needed to create attributes
+ */
+export const getIncomingRequestMetricAttributes = (
+  spanAttributes: SpanAttributes,
+  options: { component: string }
+): MetricAttributes => {
+  const metricAttributes: MetricAttributes = {};
+  metricAttributes[SemanticAttributes.HTTP_SCHEME] = options.component;
+  metricAttributes[SemanticAttributes.HTTP_METHOD] = spanAttributes[SemanticAttributes.HTTP_METHOD];
+  metricAttributes[SemanticAttributes.NET_HOST_NAME] = spanAttributes[SemanticAttributes.NET_HOST_NAME];
+  metricAttributes[SemanticAttributes.HTTP_FLAVOR] = spanAttributes[SemanticAttributes.HTTP_FLAVOR];
+  metricAttributes[SemanticAttributes.HTTP_TARGET] = spanAttributes[SemanticAttributes.HTTP_TARGET];
```

Comment:
HTTP_TARGET in span attributes now is just the path of the request, which is a high cardinality value for metrics. I'd suggest mark this as a TODO until we have a solution to parameterize this value, as suggested in https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/semantic_conventions/http-metrics.md#parameterized-attributes.

Reply:
Added TODO and removed both target and URL for the meantime.